rules/game_process.tex

```diff
@@ -33,12 +33,24 @@ \subsection{Robot States}
     So while in the \texttt{unstiff} state the robot is not allowed to move in any fashion! After booting, the robots are in their \texttt{unstiff} state.
     Pressing the chest button once while in the \texttt{unstiff} state, permits the robot to stiffen its joints and return to the \texttt{initial} state, or a state as indicated by GameController.
 
-  \item[Initial.] The robots are not allowed to be moving in any fashion besides initially standing up.
+    \item[Initial.] The robots are free to move at teams convenience and humans are allowed to interact with the robots. 
+    This state is not limited in time and teams have access to the field.
+    The GameController will activate this state before \texttt{visual} (i.e at the beginning of a half and during a timeout).
     Shortly pressing the chest button will switch the robot to the \texttt{penalized} state.
 
+    \item[Visual.] The robots are not allowed to move in any fashion besides initially standing up and moving their heads.
+    In \texttt{visual}, the robots are awaiting a visual indication from the referee which signals the transition to \texttt{ready}.  
+
   \item[Ready.] In this state, the robots walk to their legal positions for kick-off (\cf \cref{sec:kick-off}) or a penalty kick (\cf \cref{sec:penalty_kick}).
     They remain in this state, until the head referee decides that there is no significant progress, up to a maximum of \qty{\KickOffAutoTime}{\second} for a kick-off and \qty{\PenaltyKickSetupTime}{\second} for a penalty kick.
     The GameController can activate sub-states for kick-off and penalty kicks.
+    The robot can only enter this state by : 
```

Comment:
I'm still not convinced of this enumeration. Estimating the state of the game can integrate a lot of cues. For instance, why should it be illegal to enter the ready state when you (visually) observe other players entering the pitch, or (after a goal) when the ball is observed to have moved in a goal and subsequently removed? Humans certainly do this to disambiguate the state of the game and don't rely only on a sound or a signal.

I agree that there are "outside-the-box" things (such as comparing the robot's clock with the scheduled start of the match, or waiting a fixed time and such) that shouldn't be done. But my guess is that those are bad ideas that don't work well anyway. And otherwise, this is just a reiteration of the general "please don't cheat" principle.

Reply:
I removed the list. I was hesitating to move the meaning as exit condition of Visual state, but with all the instruction about the visual signal, let say strategy not involving the visual signal at all would be against the spirits of the game.